chapter04/Task_4_4.py

```diff
@@ -0,0 +1,29 @@
+# Guess the word
+#
+# The computer makes a word and answers only "Yes" or "no".
+#
+# The player has 5 attempts to guess the maximum number of letters, and then the player must write the entire word
+
+import random
+
+WORDS = ("python", "jumble", "easy", "difficult", "answer", "xylophone")
+word = random.choice(WORDS)
+
+print('In guess word', len(word), 'letters\n')
+
+i = 0
+
+while i < 5:
+    letter = input('Enter a letter\n')
+    i += 1
+    if letter in word:
```

Comment:
Should you also take ".lower()" there?
Also note that user may enter not a letter, but also a word and this would work
So probably validation that input is letter is needed.